CRM/Contact/Form/Edit/Notes.php

```diff
@@ -42,8 +42,7 @@ class CRM_Contact_Form_Edit_Notes {
    */
   public static function buildQuickForm(&$form) {
     $form->applyFilter('__ALL__', 'trim');
-    $form->add('text', 'subject', ts('Subject'), array('size' => 60, 'maxlength' => 254));
```

Comment:
are you sure this field is unused? Notes appear to have a subject.
![newnote](https://cloud.githubusercontent.com/assets/2874912/6885410/919c8e08-d5ed-11e4-8b62-9e67c88c50cd.png)